pkgs/kde/frameworks/breeze-icons/default.nix

```diff
@@ -3,14 +3,21 @@
   python3,
   libxml2,
   qtsvg,
+  buildPackages,
+  stdenv,
+  lib,
 }:
 mkKdeDerivation {
   pname = "breeze-icons";
 
-  extraNativeBuildInputs = [
-    (python3.withPackages (ps: [ ps.lxml ]))
-    libxml2
-  ];
+  extraNativeBuildInputs =
+    [
+      (python3.withPackages (ps: [ ps.lxml ]))
+      libxml2
+    ]
+    ++ lib.optional (
+      stdenv.system != buildPackages.stdenv.system
```

Comment:
Maybe use `! lib.system.equals` instead?

Reply:
`stdenv.system` is just a string, but `lib.system.equals stdenv.hostPlatform stdenv.targetPlatform` looks sensible. Will try to change

Reply:
Ok, feel free to disregard in that case.